### PR TITLE
Provide capabilities via initialstate

### PR DESCRIPTION
**Pull request by rullzer**
Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

File: lib/private/Template/JSConfigHelper.php [hunk 1]
```diff
@@ -305,6 +305,7 @@ public function getConfig() {
 		}
 
 		$this->initialStateService->provideInitialState('core', 'config', $config);
+		$this->initialStateService->provideInitialState('core','capabilities', $capabilities);
```
Comment (georgehrke) [inline lib/private/Template/JSConfigHelper.php:308]: Missing space between `'core',` and `'capabilities'`

**Review by georgehrke — APPROVED** [review]

**Review by juliusknorr — CHANGES_REQUESTED** [review]
Let's also use it in https://github.com/nextcloud/server/blob/b9bc2417e7a8dc81feb0abe20359bedaf864f790/core/src/OC/capabilities.js#L22 then right away :wink:

**Comment by rullzer** [comment]
> Let's also use it in
> 
> https://github.com/nextcloud/server/blob/b9bc2417e7a8dc81feb0abe20359bedaf864f790/core/src/OC/capabilities.js#L22
> then right away wink

Actually all part of step 2 to have a proper @nextcloud package for this

**Review by juliusknorr — APPROVED** [review]
Fine by me then :+1:

**Review by ChristophWurst — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**